unified-prototype/unified-plugin/plugin-swift/src/main/java/org/gradle/api/experimental/swift/HasSwiftTarget.java

```diff
@@ -0,0 +1,10 @@
+package org.gradle.api.experimental.swift;
+
+import org.gradle.api.provider.Property;
+import org.gradle.declarative.dsl.model.annotations.Restricted;
+
+@Restricted
+public interface HasSwiftTarget {
```

Comment:
🤔 I don't think this name makes sense.  The target is implicit and not visible in the DSL, the name makes it sound like an implementing type would have a `getTarget()` method defined (which would then hold the version property).

Could we call it `SwiftProject` or `CompilesToSwift` or something instead?

Reply:
The target is absolutely both explicit and visible in the DSL, as the `swiftVersion` property.

There's a bunch of interfaces with this naming scheme, we can do a pass over them and improve their names.